documentation/modules/ref-plan-sizing.adoc

```diff
@@ -0,0 +1,53 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-configuration-sizing-guide.adoc
+//
+[id='plan-sizing-{context}']
+= Plan sizing
+
+Plans enable dynamic scaling in the `standard` address space, as shown in the broker and router sizing sections. At the cluster level, the combination of plans and infrastructure configs will determine the max number of pods that can be deployed on the cluster. At present, {ProductName} does not support limiting the number of address spaces that gets created, so some policy on who is allowed to create them should be applied. This can be handled through standard {KubePlatform} policies.
+
+From a capacity planning perspective, it is useful to understand what the maximum number of pods and memory can be consumed for a given address space.
+
```

Comment:
Suggest splitting this out as a separate procedure file that is referenced in this reference file

Reply:
Lets do that in a separate PR